react/helpers/useInstance.jsx

```diff
@@ -1,11 +1,19 @@
 import { useState, useEffect } from 'react'
 
+export const useInstanceState = {
```

Comment:
on pourrait simplement appeler cet object `states` , `useXXX` est un mot clé pour les hooks, et ce n'en est pas un ici. Et d'ailleurs, je ne sais pas si avoir un object est utile, pourquoi pas faire du `setState('loaded')` directement, je crois que c'est comme ça qu'on fait dans cozy-client

Reply:
I check on cozy-client they are using plain text status so also I do the same